dvc/ui/__init__.py

```diff
@@ -260,6 +262,26 @@ def isatty(self) -> bool:
 
         return sys.stdout.isatty()
 
+    def open_browser(self, file: "PathLike"):
+        import webbrowser
+        from pathlib import Path
+        from platform import uname
+
+        if "Microsoft" in uname().release:
+            url = str(Path(file).relative_to(Path.cwd()))
```

Comment:
Let's `resolve` both of them anyway. Also, for `relative_to` to work, [the `cwd` needs to be parent path](https://docs.python.org/3/library/pathlib.html#id6), so we should be using `dvc.utils.relpath`/`os.path.relpath` here.

```suggestion
            path = Path(file).resolve()
            url = relpath(path) if "Microsoft" in uname().release else path.uri()
```